### PR TITLE
Answer diagnostic

### PR DESCRIPTION
**Pull request by seatuna**

### CODE REVIEW

File: lib/diagnostic.md [hunk 1]
```diff
@@ -12,7 +15,7 @@ Using the diagram in the `README` file (also in the `assets` folder), what are t
 you could return ALL list items (`li`) using jQuery? This is intentionally vague. (Assume starting at the `OL` DOM node).
 
 <!-- your answer starts here -->
-
+$('#ol').children()
 <!-- your answer ends here -->
```
Comment (RealWeeks) [inline lib/diagnostic.md:19]: TWO WAYS

File: lib/diagnostic.md [hunk 2]
```diff
@@ -62,5 +74,5 @@ item elements) to the body DOM node, then hides it.
 Use jQuery to register a click handler for clicks inside `div` items.
 
 <!-- your answer starts here -->
-
+$('#div').click(.hide())
 <!-- your answer ends here -->
```
Comment (RealWeeks) [inline lib/diagnostic.md:78]: .click is deprecated.  but this is currently correct be sure to read the docs.

**Comment by RealWeeks** [comment]
7.5/8.  jQuery pro.  Very good job.  Some of your classmates are struggling with it.  You clearly know what you're doing.  I'd encourage you to be a "beacon" on this topic and volunteer to help people if you see they need it on jQuery.

**Comment by seatuna** [comment]
( O_o )

i don't feel like a pro

**Comment by RealWeeks** [comment]
You're doing great!

**Pull request closed without merging**